### PR TITLE
Add release notes for packaging log in tmux

### PR DESCRIPTION
**Pull request by jkonecny12**
It's related to https://github.com/rhinstaller/anaconda/pull/3472 from external contributor.

### CODE REVIEW

File: docs/release-notes/tmux_packaging_log.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+:Type: TUI
+:Summary: Add packaging log to ``tmux`` tabs
+
+:Description:
+    We already have other important logs in the pre-defined tabs in tmux attached on TTY1.
+    Add the last one missing to easily get access to information about the package installation.
```
Comment (poncovka) [inline docs/release-notes/tmux_packaging_log.rst:6]: I don't think this text describes the features. Also, this is not the last one missing.

**Comment by jkonecny12** [comment]
**UPDATED:**
* Tried to improve the description. Hopefully, it's more meaningful now.

File: docs/release-notes/tmux_packaging_log.rst [hunk 2]
```diff
@@ -0,0 +1,11 @@
+:Type: TUI
+:Summary: Add packaging log to ``tmux`` tabs
+
+:Description:
+    Add the new tab to the ``tmux`` starting the Anaconda installer. This new tab will follow
+    the ``/tmp/packaging.log`` log file.
```
Comment (VladimirSlavik) [inline docs/release-notes/tmux_packaging_log.rst:6]: Just a bit more...

```suggestion
    Add a new tab to the ``tmux`` session starting the Anaconda installer. This new tab follows
    the ``/tmp/packaging.log`` log file.
```

Comment (jkonecny12) [reply docs/release-notes/tmux_packaging_log.rst]: Fixed.

**Comment by jkonecny12** [comment]
**UPDATED:**
* Changed message according to @VladimirSlavik suggestions. Thanks!

**Review by poncovka — APPROVED** [review]

**Pull request merged**